lib/features/poscan_putobject/presentation/widgets/edit_property_dialog.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:threedpass/core/widgets/buttons/text_button.dart';
+import 'package:threedpass/core/widgets/dialog/d3p_platform_dialog.dart';
+import 'package:threedpass/features/poscan_putobject/bloc/poscan_putobject_cubit.dart';
+import 'package:threedpass/features/poscan_putobject/domain/entities/poscan_property.dart';
+import 'package:threedpass/features/poscan_putobject/presentation/widgets/property_max_value.dart';
+import 'package:threedpass/features/poscan_putobject/utils/multiple_of_ten_property_validator.dart';
+
+class EditPropertyDialog extends StatelessWidget {
+  final TextEditingController _maxValueController = TextEditingController();
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  PoscanProperty prop;
+
+  EditPropertyDialog({required this.prop, final Key? key}) : super(key: key);
+
+  @override
+  Widget build(final BuildContext context) {
+    _maxValueController.text = prop.propValue.maxValue.toString();
+    return D3pPlatformDialog(
+      title: 'edit_action'.tr() +
```

Comment:
I think arguments shoule be used instead of string concatenation. For example `delete_snapshots_dialog_content` localization key has value with arguments